libs/state-machine/src/lib/getStepList.ts

```diff
@@ -0,0 +1,12 @@
+import type { WizardLayout, WizardStepLayout } from './types';
+
+export type StepListing = {
```

Comment:
If we also include the "step" here, can we allow the user to click on one of the breadcrumbs in order to move straight to that step?